src/dataTable/index.jsx

```diff
@@ -36,6 +36,9 @@ const DataTable = ({
   searchable = true,
   hasPagination = true,
   onDoubleClick,
+  isRowActive = () => {
```

Comment:
let's revisit this on Monday, will merge for now to enable testing

Reply:
sure